integration/src/test/scala/org/knora/webapi/core/LayersTest.scala

```diff
@@ -114,7 +115,9 @@ object LayersTest {
       ConstructTransformer.layer,
       DspIngestClientLive.layer,
       GravsearchTypeInspectionRunner.layer,
-      GroupsResponderADMLive.layer,
+      GroupsResponderADM.layer,
+      GroupsRestService.layer,
+      GroupService.layer,
```

Comment:
**issue**: These `Group*Service` layer invocations belong in the respective `Modules`

See also `LayersLive`

Reply:
Please see above reply.